OpenRA.Mods.Common/Traits/Cargo.cs

```diff
@@ -317,6 +320,10 @@ public void Load(Actor self, Actor a)
 
 		public void Killed(Actor self, AttackInfo e)
 		{
+			if (Info.EjectOnDeath)
+				while (!IsEmpty(self) && CanUnload())
```

Comment:
The way it is now not really anymore. I initially wanted to support parachutes as well (see EjectOnDeath). Will change it